initial pitr draft (#2753)
timescale/docs: use-timescale/backup-restore/point-in-time-recovery.md

---
title: Point-in-time recovery
excerpt: Learn how to restore your service to a previous point in time
products: [cloud]
keywords: [restore, recovery, pitr]
tags: [recovery, restore, pitr]
---

# Point-in-time recovery

Timescale allows you to perform a point-in-time recovery of your service to any 
point in the last 3 days. Point-in-time recovery allows you to recover your 

> database from a destructive or unwanted action or change. Initiating a 

Reviewer: Should the first two sentences be switched?  First describe PITR, then how Timescale's PITR works

> point-in-time recovery of your service creates a fork of your service that will 
> match the state of your database at the specified time of recovery. The 

Reviewer: ```suggestion
point-in-time recovery of your service creates a fork of your service that
matches the state of your database at the specified time of recovery. The 
```

> original service will not be deleted to avoid losing data created since the 

Reviewer: ```suggestion
original service stays untouched to avoid losing data created since the 
```

time of recovery. 

Since the point-in-time recovery is done in a fork, in order to migrate your 

> application to the point of recovery you will need to switch out the connection 

Reviewer: ```suggestion
application to the point of recovery, switch out the connection 
```
